Sources/Private/LayerContainers/AnimationContainer.swift

```diff
@@ -144,6 +144,10 @@ final class AnimationContainer: CALayer, RootAnimationLayer {
 
   var animationLayers: ContiguousArray<CompositionLayer>
 
+  var animatingLayers: [CALayer] {
+    Array(animationLayers)
+  }
```

Comment:
Could we document why we have both `animationLayers` and `animatingLayers`?

Reply:
This was because `RootAnimationLayer` has a `var animationLayers: [CALayer]` requirement but the actual layer implementations have different concrete types / subclasses for those arrays. Underscoring this shadow accessor to `var _animationsLayers: [CALayer]` would probably me more idiomatic / less confusing.